frog/imports/client/App/TopBar.js

```diff
@@ -110,34 +110,48 @@ class LogoutMenu extends React.Component<*, *> {
   }
 }
 
+type RouteT = {
+  name: string,
+  to: string[]
+};
 class TopBarController extends React.Component<
   { classes: any, history: any, location: any },
-  {}
+  { active: string }
 > {
-  routes = [
-    { name: 'Graph Editor', to: '/teacher/graph' },
-    { name: 'Sessions', to: '/t' },
+  routes: RouteT[] = [
+    { name: 'Graph Editor', to: ['/teacher/graph'] },
+    { name: 'Sessions', to: ['/teacher/orchestration', '/t'] },
     {
       name: 'Activity Preview',
-      to: '/teacher/preview'
+      to: ['/teacher/preview']
     }
   ];
 
-  value = '/teacher/preview';
+  splitPath = this.props.location.pathname.split('/');
```

Comment:
This value will not be update if the `props` change. Make sure this is intentional